InvoiceReader.Application/Queries/GetInvoices/Model.cs

```diff
@@ -9,9 +10,9 @@ public record Result
         {
             public List<Invoice> Data { get; set; }
```

Comment:
One more thing, can you please use init set, insisted of the public set.
Public set could break the DDD principles for our model.

Reply:
Yes, I will.